src/Uno.Extensions.Reactive/Core/ListState.Extensions.cs

```diff
@@ -122,6 +122,114 @@ public static ValueTask AddAsync<T>(this IListState<T> state, T item, Cancellati
 	public static ValueTask RemoveAllAsync<T>(this IListState<T> state, Predicate<T> match, CancellationToken ct = default)
 		=> state.UpdateDataAsync(itemsOpt => itemsOpt.Map(items => items.RemoveAll(match)), ct);
 
+
+	/// <summary>
+	/// Updates all items from a list state that match the key of <paramref name="oldItem"/>.
+	/// </summary>
+	/// <typeparam name="T">The type of the items in the list.</typeparam>
+	/// <param name="state">The list state onto which the item should be added.</param>
+	/// <param name="oldItem">The old value of the item.</param>
+	/// <param name="newItem">The new value for the item.</param>
+	/// <param name="ct">A token to abort the async add operation.</param>
```

Comment:
nit: the `ct` param comment mentions it is for aborting the **add** operation. Probably just a copy-paste thing but this isn't necessarily an add operation. We could be more general and just mention it as an `async operation`

Reply:
Thanks fixed